AsyncDisplayKit/ASCellNode.mm

```diff
@@ -125,6 +125,8 @@ - (void)__setNeedsLayout
 {
   CGSize oldSize = self.calculatedSize;
   [super __setNeedsLayout];
+  
+  ASDN::MutexLocker l(_propertyLock);
```

Comment:
It's for the didRelayoutFromOldSize call. I'm not sure it's necessary, it's just that a lock was held previously when __setNeedsLayout was called so I didn't want to change anything. I can add a comment to this effect?